### ISSUE TITLE
Replying to a very long message crashes the app

### ISSUE THREAD

**Issue by everypizza1** [element-hq/element-x-android#3772]
### Steps to reproduce

1. Send a very long message 
2. Attempt to reply to it

#### What did you expect?
App shouldn't crash
#### What happened instead?
App does crash 

### Your phone model

Google Pixel 8a

### Operating system version

Android 15

### Application version and app store

Element X 0.7.2 Google Play

### Homeserver

sulian.eu/Synapse 1.118

### Will you send logs?

Yes

### Are you willing to provide a PR?

No

**Comment by jmartinesp** [comment]
Can you provide some more info? I tried replying to a lorem ipsum of 22594 characters and it worked just fine.

In the stacktrace I see:

> java.lang.IllegalArgumentException: Can't represent a width of 302655 and height of 0 in Constraints

Could it be that all of the text was in a single line?

**Comment by jmartinesp** [comment]
> Could it be that all of the text was in a single line?

Yes, that did it. I believe this is a jetpack compose issue (the UI library in modern Android apps) since there were some previous crashes with similar causes that were fixed. I'm not sure if we can work around this somehow.
